ci-operator/templates/openshift/openshift-azure/build-azure-image.yaml

```diff
@@ -0,0 +1,313 @@
+# This template is being used for testing OpenShift Ansible image builds for Azure.
+# At the moment it cannot be reused to launch cluster for Origin tests
+kind: Template
+apiVersion: template.openshift.io/v1
+
+parameters:
+- name: JOB_NAME_SAFE
+  required: true
+- name: JOB_NAME_HASH
+  required: true
+- name: NAMESPACE
+  required: true
+- name: CLUSTER_TYPE
+  value: "azure"
+  required: true
+- name: AZURE_VM_SIZE
+  value: "Standard_D2s_v3"
+  required: true
+- name: OPENSHIFT_RELEASE
+  value: "3.10"
+  required: true
+- name: IMAGE_ANSIBLE
```

Comment:
I think if we use ansible release spec now this is not needed and we need to use IMAGE_SRC ?

Reply:
No, I think we still need the built image, because src is just the openshift-ansible repo w/o any tools? Either way, this is going to run as a periodic for now and there is a separate card for enabling it to work for o-a PRs.

Reply:
IMAGE_BIN? as it should be build as part of release-3.10.json config? As by specifying IMAGE_ANSIBLE we dont use the image build on that particular PR, we use latest promoted image from origin-3.10 namespace

Reply:
Still fine, this is going to run as a periodic for now.

Reply:
otherwhise what value we get by pointing to openshift-ansible config if we  dont use result of it.

Reply:
Making it work for o-a PRs is https://jira.coreos.com/projects/AZURE/issues/AZURE-40

This designs gets us closer to that.